explainaboard/processors/argument_pair_extraction.py

```diff
@@ -0,0 +1,273 @@
+            sentences = output["sentences"]
+            true_spans, pred_spans = self._block_ops.get_blocks(
+                output['true_tags'], output['pred_tags'], sentences
+            )
+
+            # merge the spans together
+            merged_spans: dict[tuple[int, int], Block] = {}
+            for span in true_spans:
+                span.block_tag = f'{span.block_tag} {self._DEFAULT_TAG}'
+                merged_spans[unwrap(span.block_pos)] = span
+            for span in pred_spans:
+                merged_span = merged_spans.get(unwrap(span.block_pos))
+                if not merged_span:
+                    span.block_tag = f'{self._DEFAULT_TAG} {span.block_tag}'
+                    merged_spans[unwrap(span.block_pos)] = span
+                else:
+                    true_tag, _ = unwrap(merged_span.block_tag).split(' ')
+                    merged_span.block_tag = f'{true_tag} {span.block_tag}'
+            # analysis cases
+            for ms in merged_spans.values():
+                true_tag, pred_tag = unwrap(ms.block_tag).split(' ')
+                case = AnalysisCaseLabeledBlock(
+                    sample_id=i,
+                    features={},
+                    text=unwrap(ms.block_text),
+                    true_label=true_tag,
+                    predicted_label=pred_tag,
+                    block_review_sentences=unwrap(ms.block_review_sentences),
+                    block_review_tokens=unwrap(ms.block_review_tokens),
+                    block_review_position=unwrap(ms.block_review_position),
+                    block_reply_sentences=unwrap(ms.block_reply_sentences),
+                    block_reply_tokens=unwrap(ms.block_reply_tokens),
+                    block_reply_position=unwrap(ms.block_reply_position),
+                    orig_str="source",
+                )
+                for feat_name, feat_spec in analysis_level.features.items():
+                    if feat_spec.func is None:
+                        raise ValueError(
+                            f'could not find feature function for {feat_name}'
+                        )
+                    elif not feat_spec.require_training_set:
+                        case.features[feat_name] = feat_spec.func(
+                            sys_info, output, case
+                        )
+                    elif statistics is not None:
+                        case.features[feat_name] = feat_spec.func(
+                            sys_info, output, case, statistics
+                        )
+                cases.append(case)
+        # calculate metric stats
+        true_data = [x.true_label for x in cases]
+        pred_data = [x.predicted_label for x in cases]
+        metric_stats: list[MetricStats] = [
+            x.to_metric().calc_stats_from_data(true_data, pred_data)
+            for x in analysis_level.metric_configs
+        ]
+        return cast(List[AnalysisCase], cases), metric_stats
```

Comment:
It will following errors if we don't cast
```shell
xplainaboard/processors/argument_pair_extraction.py:228: error: "AnalysisCase" has no attribute "true_label"
explainaboard/processors/argument_pair_extraction.py:229: error: "AnalysisCase" has no attribute "predicted_label"
```